README.md

```diff
@@ -1,23 +1,23 @@
-# TV Kitchen Appliance Core
+[![Build Status](https://travis-ci.com/tvkitchen/base.svg?branch=master)](https://travis-ci.com/tvkitchen/base)
```

Comment:
Whoops, copypasta! Needs to point to `tvkitchen/appliances`, not `tvkitchen/base`.